LICENSES.md

```diff
@@ -0,0 +1,7 @@
+# Licenses for special features
```

Comment:
Too many unnecessary changed files and commits, may close this PR and re-create one.